include/picolibrary/microchip/megaavr0/peripheral/clkctrl.h

```diff
@@ -579,6 +901,339 @@ class CLKCTRL {
+     */
+    void set_internal_32_768_khz_ultra_low_power_oscillator_mode( Internal_32_768_kHz_Ultra_Low_Power_Oscillator_Mode mode ) noexcept
+    {
+        return osc32kctrla.set_internal_32_768_khz_ultra_low_power_oscillator_mode( mode );
+    }
+
+    /**
+     * \copydoc picolibrary::Microchip::megaAVR0::Peripheral::CLKCTRL::MCLKSTATUS::external_32_768_khz_crystal_oscillator_stable()
+     */
+    auto external_32_768_khz_crystal_oscillator_stable() const noexcept
+    {
+        return mclkstatus.external_32_768_khz_crystal_oscillator_stable();
+    }
+
+    /**
+     * \copydoc picolibrary::Microchip::megaAVR0::Peripheral::CLKCTRL::XOSC32KCTRLA::external_32_768_khz_crystal_oscillator_source()
+     */
+    auto external_32_768_khz_crystal_oscillator_source() const noexcept
+    {
+        return xosc32kctrla.external_32_768_khz_crystal_oscillator_source();
+    }
+
+    /**
+     * \copydoc picolibrary::Microchip::megaAVR0::Peripheral::CLKCTRL::XOSC32KCTRLA::external_32_768_khz_crystal_oscillator_start_up_time()
+     */
+    auto external_32_768_khz_crystal_oscillator_start_up_time() const noexcept
+    {
+        return xosc32kctrla.external_32_768_khz_crystal_oscillator_start_up_time();
+    }
+
+    /**
+     * \copydoc picolibrary::Microchip::megaAVR0::Peripheral::CLKCTRL::XOSC32KCTRLA::external_32_768_khz_crystal_oscillator_mode()
+     */
+    auto external_32_768_khz_crystal_oscillator_mode() const noexcept
+    {
+        return xosc32kctrla.external_32_768_khz_crystal_oscillator_mode();
+    }
+
+    /**
+     * \copydoc picolibrary::Microchip::megaAVR0::Peripheral::CLKCTRL::XOSC32KCTRLA::configure_external_32_768_khz_crystal_oscillator()
+     */
+    void configure_external_32_768_khz_crystal_oscillator(
+        External_32_768_kHz_Crystal_Oscillator_Source        source,
+        External_32_768_kHz_Crystal_Oscillator_Start_Up_Time start_up_time,
+        External_32_768_kHz_Crystal_Oscillator_Mode          mode ) noexcept
+    {
+        xosc32kctrla.configure_external_32_768_khz_crystal_oscillator( source, start_up_time, mode );
+    }
+
+    /**
+     * \copydoc picolibrary::Microchip::megaAVR0::Peripheral::CLKCTRL::OSC32KCTRLA::set_external_32_768_khz_crystal_oscillator_mode()
+     */
+    void set_external_32_768_khz_crystal_oscillator_mode( External_32_768_kHz_Crystal_Oscillator_Mode mode ) noexcept
+    {
+        return xosc32kctrla.set_external_32_768_khz_crystal_oscillator_mode( mode );
+    }
+
+    /**
+     * \copydoc picolibrary::Microchip::megaAVR0::Peripheral::CLKCTRL::XOSC32KCTRLA::external_32_768_khz_crystal_oscillator_enabled()
+     */
+    auto external_32_768_khz_crystal_oscillator_enabled() const noexcept
+    {
+        return xosc32kctrla.external_32_768_khz_crystal_oscillator_enabled();
+    }
+
+    /**
+     * \copydoc picolibrary::Microchip::megaAVR0::Peripheral::CLKCTRL::OSC32KCTRLA::enable_external_32_768_khz_crystal_oscillator()
```

Comment:
`\copydoc` reference is incorrect.
```suggestion
     * \copydoc picolibrary::Microchip::megaAVR0::Peripheral::CLKCTRL::XOSC32KCTRLA::enable_external_32_768_khz_crystal_oscillator()
```